frontend/src/views/contributions.js

```diff
@@ -32,46 +33,28 @@ export const ContributionsPage = props => {
 
   return (
     <>
-    <div className="pt4-l pb5 ph5-l ph2 pt180 pull-center bg-tan">
-      {
-        props.children
-        /* This is where the full task body component is rendered
+      <div className="pt4-l pb5 ph5-l ph2 pt180 pull-center bg-tan">
+        {
+          props.children
+          /* This is where the full task body component is rendered
         using the router, as a child route.
         */
-      }
-      <section className="cf">
-
-        <MyTasksNav />
-        <TaskResults retryFn={forceUpdate} state={state} />
-        {/* TODO support pagination on this API
+        }
+        <section className="cf">
+          <MyTasksNav />
+          <TaskResults retryFn={forceUpdate} state={state} />
+          {/* TODO support pagination on this API
         <ProjectCardPaginator projectAPIstate={state} setQueryParam={setContributionsQuery} />
         */}
 
-        {/* delete me! TDK */}
-        <code className={`dn`}>{JSON.stringify(state)}</code>
-      </section>
-    </div>
+          {/* delete me! TDK */}
+          <code className={`dn`}>{JSON.stringify(state)}</code>
```

Comment:
@willemarcel please delete these two lines. It accidentally got in from notifications and creates a lot of DOM junk.